app/views/shared/_event_logs_table.html.erb

```diff
@@ -8,8 +8,8 @@
     ],
     rows: logs.map do |log|
     [ { text: formatted_date(log), format: "event-log-date" },
-      { text: formatted_message(log) } ]
-    end
+      { text: formatted_message(log) } ] unless log.requires_admin? && !user.govuk_admin?
```

Comment:
I think we can just call `&& !current_user.govuk_admin?` here and remove the passing of the user local to the views here, actually. I think it'd read a bit more clearly as it's unclear which user this is - is it the user being viewed or the user doing the viewing. I think this is where our confusion came from yesterday.

Reply:
Sorted both, this makes perfect sense. Got a bit lost with the `user` confusion to notice this